hr_job_categories/tests/test_hr_job_categories.py

```diff
@@ -25,34 +25,53 @@
 class test_hr_job_categories(common.TransactionCase):
     def setUp(self):
         super(test_hr_job_categories, self).setUp()
```

Comment:
please, use the v8 api in the tests. self.registry(...) -> self.env[...] and remove parameters cr, uid, context